config/dev.exs

```diff
@@ -4,3 +4,5 @@ config :elix, Elix.Robot,
   adapter: Hedwig.Adapters.Console
 
 config :elix, :api_client, Elix.APIClient
+
+config :elix, :redis_client, Redix
```

Comment:
Since mocks will only ever be used in the test environment, I think it makes sense to declare the actual implementation in `config.exs` so it's inherited by all other environments, rather than having to specify it individually in `dev`, `prod`, etc.